NineChronicles.Headless/GraphTypes/StandaloneQuery.cs

```diff
@@ -72,6 +74,87 @@ public StandaloneQuery(StandaloneContext standaloneContext, IConfiguration confi
                 }
             );
 
+            Field<ListGraphType<RootStateDiffType>>(
+                name: "diffs",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<LongGraphType>>
+                    {
+                        Name = "baseIndex",
+                        Description = "The index of the block used to fetch state from chain."
+                    },
+                    new QueryArgument<NonNullGraphType<LongGraphType>>
+                    {
+                        Name = "changedIndex",
+                        Description = "The index of the block used to fetch state from chain."
+                    }
+                ),
+                resolve: context =>
+                {
+                    if (!(standaloneContext.BlockChain is BlockChain blockChain))
+                    {
+                        throw new ExecutionError(
+                            $"{nameof(StandaloneContext)}.{nameof(StandaloneContext.BlockChain)} was not set yet!"
+                        );
+                    }
+
+                    var baseIndex = context.GetArgument<long>("baseIndex");
+                    var changedIndex = context.GetArgument<long>("changedIndex");
+
+                    var blockInterval = changedIndex - baseIndex;
+
+                    if (blockInterval >= 10 && blockInterval > 0)
```

Comment:
Maybe it means:

```suggestion
                    var blockInterval = Math.abs(changedIndex - baseIndex);

                    if (blockInterval >= 10)
```

Reply:
I figured that if `baseIndex` is higher than `changedIndex`, it’s a mistake.
So, I made sure the validation doesn’t allow negatives.

Reply:
https://github.com/planetarium/NineChronicles.Headless/pull/2479#discussion_r1609426393
My bad, I thought negative values were not acceptable.

Reply:
Applied, thanks